fluxacademy.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<head>
+	<title>Flux Academy</title>
+</head>
+<body>
+    <img src="">
+    //sticky nav how to
+    <nav>
+        <ul>
+            <img src="" alt="logo">
```

Comment:
Only `<li>` elements can be children of `<ul>`
Move `<img>` out of here and make it a direct child of `<nav>`